ci/commoncmd.sh

```diff
@@ -1,3 +1,2 @@
 #! /bin/sh
-yarn install --frozen-lockfile
 yarn run build
```

Comment:
I don't think you need to build stuff every time - IIRC only the mozilla job needs it